### PR TITLE
Drop "value" parameter from Task._step method.

### PR DESCRIPTION
**Pull request by 1st1**

### CODE REVIEW

File: asyncio/tasks.py [hunk 1]
```diff
@@ -231,12 +231,12 @@ def _step(self, value=None, exc=None):
         self._fut_waiter = None
 
         self.__class__._current_tasks[self._loop] = self
-        # Call either coro.throw(exc) or coro.send(value).
+        # Call either coro.throw(exc) or coro.send(None).
         try:
-            if exc is not None:
-                result = coro.throw(exc)
+            if exc is None:
+                result = coro.send(None)
```
Comment (gvanrossum) [inline asyncio/tasks.py]: Shouldn't this be next(coro)?

Comment (1st1) [inline asyncio/tasks.py]: No, `async def` coroutines don't have `__iter__` and `__next__`...

Comment (gvanrossum) [inline asyncio/tasks.py]: That deserves a comment then!

Otherwise, LGTM.

**Comment by 1st1** [comment]
Merged by hand. @gvanrossum Thanks for the review! Here's the comment with an explanation of why not `next(coro)`: https://github.com/1st1/asyncio/commit/9b9ac203393c8fbd9ba575cf61cd52d76f8869cc

**Pull request closed without merging**